server/db/repository/UserRepository.js

```diff
@@ -3,31 +3,31 @@ const User = require('../models/User');
 
 async function getUserByObjectId(id) {
   try {
-    return User.findOne({ _id: id });
+    return User.findOne({ _id: id }).exec();
```

Comment:
why the exec now?

Reply:
https://mongoosejs.com/docs/promises.html#queries-are-not-promises